docs/source/kinds_of_types.rst

```diff
@@ -489,8 +489,8 @@ defined. Thus this code does not work as expected:
    class A:
        ...
 
-In cases like these you can enter the type as a string literal — this
-is a *forward reference*:
+In cases when a type hint contains names that have not been defined yet,
```

Comment:
I think the previous wording is fine to keep.

Reply:
Okay, I'll keep previous